### PR TITLE
Add headers and libraries for Tizen device.

### PR DESCRIPTION
**Pull request by pyrasis**
- Add headers and libraries for Tizen device.
-  Add libwebsockets headers and library

### CODE REVIEW

File: CocosDenshion/proj.tizen/.project [hunk 1]
```diff
@@ -30,7 +31,7 @@
 				</dictionary>
 				<dictionary>
 					<key>org.eclipse.cdt.make.core.buildLocation</key>
-					<value>${workspace_loc:/cocosdenshion/Debug-Tizen-Emulator}</value>
+					<value>${workspace_loc:/cocosdenshion/Debug-Tizen-Device}</value>
```
Comment (dumganhar) [inline CocosDenshion/proj.tizen/.project:34]: Did you get a tizen device for testing?

**Comment by pyrasis** [comment]
I'm sorry.
I found some problem. so, I will pull request again.

**Comment by dumganhar** [comment]
OK. Could you also send the changes of tizen port for `master` branch?

**Comment by pyrasis** [comment]
OK. I will work on master branch.

**Comment by dumganhar** [comment]
Hi @pyrasis 
We are preparing cocos2d-x 2.1.4 this week. So if you have some changes for tizen, please send to us immediately.
Thanks.

**Comment by pyrasis** [comment]
@dumganhar 
Sorry. I will not be able to complete this work by this week unfortunately.
I want to add this to the next version.

**Comment by dumganhar** [comment]
@pyrasis 
OK, so we just keep the current state of tizen port on master branch for this release.
BTW, could you also test tizen port?
Do you mind to let me know your email address for farther connection?
My email is `jianhua.chen@cocos2d-x.org`
Thanks.

**Comment by pyrasis** [comment]
@dumganhar 
Sure, I can test tizen port.
My email is pyrasis@gmail.com
Thanks.

**Comment by dumganhar** [comment]
@pyrasis 
Thanks. :)

**Pull request closed without merging**